build.sh

```diff
@@ -1,20 +1,129 @@
 #!/bin/bash
-set -x
-set -e
 
-echo Running basic sanity tests on metadata.
-echo If these fail, then fix whatever\'s causing them first.
- 
-prove
+KSP_VERSION_DEFAULT="1.0.2"
+KSP_NAME_DEFAULT="dummy"
 
+# Function for creating dummy KSP directories to test on. Takes version as an argument.
+create_dummy_ksp () {
+    # Set the version to the requested KSP version if supplied.
+    _KSP_VERSION=$KSP_VERSION_DEFAULT
+    _KSP_NAME=$KSP_NAME_DEFAULT
+    
+    if [ $# -eq 2 ]
+    then
+        _KSP_VERSION=$1
+        _KSP_NAME=$2
+    fi
+    
```

Comment:
```
Couldn't find install with name "579782bb95bbfe990e7c8bb602d7262564ae8cd4", aborting..
```

I've not yet diagnosed that if block. But setting the thing to ```_KSP_NAME``` to ```$KSP_NAME``` caused it to set the install name correctly.